src/bundle/Resources/public/scss/_instant-filter.scss

```diff
@@ -8,21 +8,33 @@
 .ez-extra-actions--create {
     .ez-instant-filter {
         &__group-name {
-            padding-left: .5rem;
+            padding-left: calculateRem(8px);
         }
 
         &__input-wrapper {
-            padding: 1rem .5rem;
+            padding: calculateRem(16px) calculateRem(8px);
         }
 
         &__group-item {
+            position: relative;
+
             label {
                 cursor: pointer;
             }
+
+            .ez-icon {
+                fill: $ez-black;
+                position: absolute;
+                top: 0;
+                left: calculateRem(20px);
+                width: calculateRem(16px);
+                height: calculateRem(16px);
```

Comment:
You should use `ez-icon--small`

Reply:
I defined it here in order to overwrite rule: `.ez-context-menu .ez-sticky-container .ez-icon`.